config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/WSDL.java

```diff
@@ -1,11 +1,15 @@
 package com.ca.apim.gateway.cagatewayconfig.beans;
 
-public class WSDL {
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public class WSDL extends Folderable {
```

Comment:
please add javadoc to the class

and according to the google code standards we are going to standardize, only first letter is capital
so could you rename everything from WSDL to Wsdl?